Flat/src/ModulesLepPho.cc

```diff
@@ -9,139 +9,262 @@
 using namespace panda;
 using namespace std;
 
+void PandaAnalyzer::SimpleLeptons() {
+  looseLep1PdgId=-1, looseLep2PdgId=-1;
+  //electrons
+  for (auto& ele : event.electrons) {
+    float pt = ele.pt(); float eta = ele.eta(); float aeta = fabs(eta);
+    if (pt<10 || aeta>2.5) continue;
+    if (!ele.veto) continue;
+    if (!ElectronIP(ele.eta(),ele.dxy,ele.dz)) continue;
+    ele.setPtEtaPhiM(pt,eta,ele.phi(),511e-6);
+    unsigned iL=gt->nLooseElectron;
+    bool isFake   = ele.hltsafe;
+    bool isMedium = ele.medium;
+    bool isTight  = ele.tight;
+    if (isTight) gt->nTightElectron++;
+    int eleSelBit            = kLoose;
+    if (isFake  ) eleSelBit |= kFake;
+    if (isMedium) eleSelBit |= kMedium;
+    if (isTight ) eleSelBit |= kTight;
+    gt->electronPt[iL]           = pt;
+    gt->electronEta[iL]          = eta;
+    gt->electronPhi[iL]          = ele.phi();
+    gt->electronSfLoose[iL]      = GetCorr(cEleLoose, eta, pt);
+    gt->electronSfTight[iL]      = GetCorr(cEleTight, eta, pt);
+    gt->electronSfUnc[iL]        = GetError(cEleMedium, eta, pt);
+    gt->electronSfReco[iL]       = GetCorr(cEleReco, eta, pt);
+    gt->electronSelBit[iL]       = eleSelBit;
+    gt->electronPdgId[iL]        = ele.charge*-11;
+    gt->electronTripleCharge[iL] = ele.tripleCharge;
```

Comment:
This shouldn't be filled here - it's not booked in the three.

Reply:
Ok, the fix here is to book it in the tree even if not using complicated leptons, will update

Reply:
Or not fill it? We don't use it in the simple lepton case.

Reply:
Agree we should drop electronTripleCharge and all scalefactor branches in favor of the single lepton sf number from PandaAnalysis::LeptonSF. However keep in mind that looseLepNPdgId- and looseLepNIsTight-type branches are gone, so we should keep electronSelBit, electronPdgId, and the muon analogues in the simple leptons case. This actually might reduce the space requirement for electrons as HLT safe and tight ID are now stored in 1 integer (electronSelBit) instead of 2.